tests/test_project/tests/base.py

```diff
@@ -24,6 +24,12 @@ def user_change_password(self, user_number, password_number):
         user.set_password(self.PASSWORD_TEMPLATE % password_number)
         user.save()
 
+    def user_change_password_iterating(self, user_number, iterations):
```

Comment:
@spokerman12, nit: missing docstring. Also, can you rename this to something like `user_change_password_multiple_times` where the second argument is named something like `times` or `number_of_times`?